src/components/layouts/EditForm/index.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import Button from '../../ui/Button';
+import EditIcon from '../../ui/EditIcon';
+import convertFromPixelsToRem from '../../../settings/__convertToCssRemUnit';
+
+interface Props {
+  handleSubmit: () => void;
+  loading?: boolean;
+  imageLink?: string;
+  displayHeader?: boolean;
+  imageName?: string;
+  submitButtonName: string;
+  children: React.ReactNode;
+}
+
+const MobileForm = (props: Props) => {
+  const {
+    handleSubmit = () => null,
+    loading = false,
+    imageLink = '',
+    displayHeader = true,
+    imageName = '',
+    submitButtonName = '',
+    children,
+  } = props;
+  return (
+    <MobileForm.Wrapper>
+      {displayHeader && (
+        <MobileForm.Header>
+          <div className="image-wrapper">
+            <img src={imageLink} className="user-image" alt={imageName} />
+            <div className="image-overlay">
+              <EditIcon />
+            </div>
+          </div>
+        </MobileForm.Header>
+      )}
+      <MobileForm.Form>
+        {children}
+        <Button isLoading={loading} handleClick={handleSubmit}>
+          {' '}
+          {submitButtonName}
+        </Button>
+      </MobileForm.Form>
+    </MobileForm.Wrapper>
+  );
+};
+
+MobileForm.Wrapper = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+MobileForm.Header = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 8.25rem;
+  .image-wrapper {
+    position: relative;
+    width: ${convertFromPixelsToRem(70)};
+    height: ${convertFromPixelsToRem(70)};
+    background: gray;
+    border-radius: 50%;
+    overflow: hidden;
+    .user-image {
+      width: 100%;
+      height: 100%;
+    }
+    .image-overlay {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 100%;
+      position: absolute;
+      background: rgba(255, 255, 255, 0.7);
+      height: 50%;
+      top: 50%;
+      right: 0;
+    }
+  }
+`;
+MobileForm.Form = styled.form`
+  width: 100%;
+  height: 100%;
+  padding: 1.5rem;
+  div {
+    button {
+      border-radius: ${convertFromPixelsToRem(3)};
+    }
+  }
+`;
+export default MobileForm;
```

Comment:
This form should be in the `components/ui` folder as it is a piece of UI.

Reply:
I thought since we are following the atomic design pattern, the components in the UI folder are those that can't be further decomposed (i.e  buttons, input, etc)

Reply:
The UI folder contains individual UI elements that are not connected to the store but display some sort of physical visuals on the screen.

Reply:
resolved.